star_tides/services/databases/mongo/models/guide_model.py

```diff
@@ -0,0 +1,22 @@
+''' star_tides.services.mongo.models.user_model
+'''
+from mongoengine import (
+    Document,
+    StringField,
+    UUIDField,
+    ListField,
+    EmbeddedDocumentField
+)
+
+from star_tides.services.databases.mongo.models.guidance_model import Guidance
+
+
+class Guide(Document):
+    ''' User model
+    '''
+    uuid = UUIDField(binary=False, required=False)
+    name = StringField(required=True)
+    focuses = ListField(required=True)
+    related_projects = ListField(required=False, default=[])
+    relevant_contracts = ListField(required=False, default=[])
+    guidance = EmbeddedDocumentField(Guidance, required=True)
```

Comment:
This will be a list of steps in a checklist so this should be a list of embedded guidances.

Think of Guides as a Todo List with some extra textual content and links etc.

Reply:
Ok I think I got it this time around.